CONTRIBUTING.md

```diff
@@ -24,14 +23,13 @@ In any case, **don't worry if you create an issue here**, we'll assist you with
 
```

Comment:
I would remove the sentence `In any case, **don't worry if you create an issue here**, we'll assist you with your problem.`